src/indexers/indexer.utils.ts

```diff
@@ -63,6 +74,106 @@ export async function vaultToSnapshot(chain: Chain, vaultDefinition: VaultDefini
   };
 }
 
+export async function constructVaultModel(
+  chain: Chain,
+  sdk: BadgerSDK,
+  vault: RegistryVault,
+): Promise<Nullable<VaultCompoundModel>> {
+  const { address, totalSupply, balance, pricePerFullShare, available } = vault;
+
+  let vaultAddr;
+
+  try {
+    vaultAddr = ethers.utils.getAddress(vault.address).toLowerCase();
```

Comment:
let's keep all entries as checksum consistent with rest of api

Reply:
gotcha, fixed